scripts/protoc-swagger-gen.sh

```diff
@@ -30,5 +30,9 @@ wget "https://raw.githubusercontent.com/cosmos/cosmos-sdk/${SDK_VERSION}/client/
 # all the individual swagger files need to be configured in `config.json` for merging
 swagger-combine ./client/docs/config.json -o ./tmp-swagger-gen/swagger.yaml -f yaml --continueOnConflictingPaths true --includeDefinitions true
 
+# move generated swagger file to swagger-ui directory
+rm -f ./client/docs/swagger-ui/swagger.yaml
```

Comment:
is this still valid or do we need to keep swagger.yaml?
```suggestion
```

Reply:
Yes, we need this file for `make update-swagger-docs`.

Reply:
we don't need this line, since we are replacing the swagger file in next command.